lib/trino-filesystem-azure/pom.xml

```diff
@@ -152,6 +152,12 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Why is it needed?

Reply:
One of the tests calls `slice.toUtf8String` so it has to be declared